data_validation/cli_tools.py

```diff
@@ -376,6 +376,12 @@ def _configure_validate_parser(subparsers):
     validate_parser.add_argument(
         "--result-handler-config", "-rc", help="Result handler config details"
     )
+    validate_parser.add_argument(
```

Comment:
I'm not sure about the binary (single value) design of this field.  Particularly, it's limiting going forward as it makes adding a third status (eg warning) difficult to fit into the design.

The value of ALL also ends up being an implicit specification throughout the entire codebase which can be difficult for readability and testing.

I would prefer if this flag allowed a comma separated list of status' and the default was `success,fail`.